core/templates/pages/exploration-player-page/learner-experience/conversation-skin-embed.directive.html

```diff
@@ -1,22 +1,11 @@
-  <!--
-  Off-screen preview of the next card in order to pre-determine the target
-  height for the card content transition animation.
-  -->
-<div class="conversation-skin-future-tutor-card" aria-hidden="true">
-  <div class="oppia-learner-view-card-content">
-    <div class="oppia-learner-view-card-top-section">
-      <angular-html-bind class="oppia-rte-viewer oppia-learner-view-card-top-content"
-                         html-data="nextCard.getContentHtml()">
-      </angular-html-bind>
-    </div>
-    <div ng-if="upcomingInlineInteractionHtml">
-      <div class="conversation-skin-inline-interaction">
-        <angular-html-bind html-data="upcomingInlineInteractionHtml">
-        </angular-html-bind>
-      </div>
-    </div>
-  </div>
-</div>
+<!--
```

Comment:
I thought you said we didn't need an off-screen preview any more? (ref: https://github.com/oppia/oppia/pull/14049#issuecomment-965307609)

Reply:
forgot to remove it from here.